### ISSUE TITLE
Emscripten example not working on chrome, v0.9.8

### ISSUE THREAD

**Issue by chuckleplant** [openframeworks/openFrameworks#5613]
I'm trying to follow this http://openframeworks.cc/setup/emscripten/ 
And everything seems to build fine, but when I `emrun --browser chrome bin/3DPrimitivesExample.html` I get an exception:

```c++
missing function: _ZNSt3__112basic_stringIcNS_11char_traitsIcEENS_9allocatorIcEEE6assignEPKc
-1
-1
warning: build with  -s DEMANGLE_SUPPORT=1  to link in libcxxabi demangling
exception thrown: abort(-1) at Error
    at jsStackTrace (http://localhost:6931/3DPrimitivesExample.js:1:23430)
    at stackTrace (http://localhost:6931/3DPrimitivesExample.js:1:23601)
    at abort (http://localhost:6931/3DPrimitivesExample.js:36:169827)
    at __ZNSt3__112basic_stringIcNS_11char_traitsIcEENS_9allocatorIcEEE6assignEPKc (http://localhost:6931/3DPrimitivesExample.js:1:279449)
    at Wec (http://localhost:6931/3DPrimitivesExample.js:20:175177)
    at jta (http://localhost:6931/3DPrimitivesExample.js:8:154433)
    at pIa (http://localhost:6931/3DPrimitivesExample.js:5:337039)
    at mIa (http://localhost:6931/3DPrimitivesExample.js:5:336675)
    at tKa (http://localhost:6931/3DPrimitivesExample.js:12:25644)
    at rKa (http://localhost:6931/3DPrimitivesExample.js:12:25412)
If this abort() is unexpected, build with -s ASSERTIONS=1 which can give more information.

```

---------
**Platform**: OSX 10.12.4
**OF**: v0.9.8
**emcc**: v1.37.9
**clang**: v4.0.0

**Comment by chuckleplant** [comment]
Just tried on latest master and got this other error:

```
ERROR:root:/Users/sergio-mb/Development/ChuckFrameworks/libs/freetype/lib/emscripten/libfreetype.bc: No such file or directory ("/Users/sergio-mb/Development/ChuckFrameworks/libs/freetype/lib/emscripten/libfreetype.bc" was expected to be an input file, based on the commandline arguments provided)
```

**Comment by chuckleplant** [comment]
Yay, master works. 

Sorry for the hassle ^^

**Comment by chuckleplant** [comment]
Self-reminder, as I'm back here every two days:

- Synchronize with upstream following [this](https://github.com/openframeworks/openFrameworks/blob/master/INSTALL_FROM_GITHUB.md)
- Run the emscripten/download_libs.sh script
- Run the emscripten instructions 
  - `./emsdk update`
  - `./emsdk install latest`
  - `./emsdk activate latest`
- Clean your project: `emmake make clean`
- Build and run on chrome!

When syncing emscripten builds crash for some reason, if not, it's the normal build (XCode) seems like the boost libraries are overwritten ❓

**Comment by arturoc** [comment]
not sure what you mean with:

```
When syncing emscripten builds crash for some reason, if not, it's the normal build (XCode) seems like the boost libraries are overwritten :question:
```

**Comment by chuckleplant** [comment]
By syncing I meant pulling upstream (main oF repo) and downloading osx libs. But you solved this in #5693 👍
